### PR TITLE
Refine user guide

### LINKED ISSUE #95: Refine user guide
- Change mentions of "Addressbook" to "HiredFiredPro"
- Fix minor typos
- Update the command summary

### PR DESCRIPTION
**Pull request by SQ77**
closes #95

### CODE REVIEW

**Review by LemonDrew — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -137,20 +137,20 @@ Format: `reject n/NAME j/JOB`
 * Changes the person's status to `Rejected`
 * Both the name of the person and the job that he/she is applying for must be provided.
 * Existing status will be updated to the `Rejected` status.
-* The search is case-insensitive for both name and job. e.g `hans` will match `Hans` and `software engineer` will match
+* The search is case-insensitive for both name and job. e.g. `hans` will match `Hans` and `software engineer` will match
```
Comment (bigismols) [inline docs/UserGuide.md:140]: Some statement have full stop and some don't. Should we standardize this?

**Review by bigismols — COMMENTED** [review]
Requested minor changes. LGTM otherwise

File: docs/UserGuide.md (hunk 1 above)
Comment (SQ77) [reply docs/UserGuide.md:140]: Ok, I have added full stops to make the formatting consistent.

**Pull request merged**